dashboard/test/controllers/pd/workshop_enrollment_controller_test.rb

```diff
@@ -30,48 +30,51 @@ class Pd::WorkshopEnrollmentControllerTest < ::ActionController::TestCase
     )
   end
 
-  test 'non-logged-in users can not enroll in csd' do
+  test 'non-logged-in users cannot enroll in csf workshop' do
+    workshop = create :pd_workshop
```

Comment:
I'd explicitly specific COURSE_CSF here and in other CSF tests.
The default course for :pd_workshop is actually `Pd::Workshop::COURSES.first`, which happens to be COURSE_CSF. However if the order of courses changes, this test will become invalid without anyone notices (it still passes though).